app/src/main/java/com/venuenext/venuenextsdkdemo/MainActivity.kt

```diff
@@ -162,18 +190,37 @@ class MainActivity : AppCompatActivity(), BottomNavigationView.OnNavigationItemS
     }
 
     private fun completeInitialize() {
+
         registerFCM()
+        VNOrderUI.initialize()
+        VNWalletUI.initialize()
+        VNPayment.configurePaymentProcessing(BraintreePaymentProcessableFragment())
+
+        VenueNext.notificationTitle = "Intel Demo"
+        VenueNext.notificationSmallIcon = R.mipmap.ic_launcher
+        VenueNext.notificationLargeIcon = R.mipmap.ic_launcher
+
+        VenueNext.configureAnalytics(FirebaseAnalytics(this@MainActivity))
+        VenueNext.configureAnalytics(LocalyticsAnalytics(this@MainActivity, application!!))
 
         bottomNavigationView.visibility = View.VISIBLE
+        navigateAfterInitialization()
 
+        // Only handle deep links after the SDK has loaded
+        handleIntent()
```

Comment:
(Aside) This is part of navigation, since the deep link always results in navigation. I don't think it would make sense if this was in `completeInitializeAfterFailure` either 🤷🏽‍♂️